lib/loaders/loaders_with_authentication/impulse.js

```diff
@@ -34,13 +32,5 @@ export default (accessToken, userID, requestIDs) => {
       },
       { method: "POST" }
     ),
-    markMessageReadLoader: impulseLoader(
```

Comment:
We already had a `conversationUpdateLoader` for loyalty that just does a `PUT` to the conversation, so we can just use that.